.github/workflows/go-scheduled.yml

```diff
@@ -30,40 +32,22 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-go-
 
-    - name: Get dependencies
-      run: |
-        go get -v -t -d ./...
-        if [ -f Gopkg.toml ]; then
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            dep ensure
-        fi
-
-    - name: Format
-      run: |
-        diff -u <(echo -n) <(gofmt -d .)
```

Comment:
Code format can be changed only via PRs, ergo there is no point in testing it periodically when we already do this pre and post merge.